integration-tests/src/test/java/oracle/kubernetes/operator/utils/TestUtils.java

```diff
@@ -1337,4 +1492,19 @@ public static void buildDeployWebServiceAppInPod(
     domain.callShellScriptToBuildDeployAppInPod(
         appName, scriptName, username, password, clusterURL, wsServiceName);
   }
+
+  public static void executeCmd(String cmd) throws Exception {
```

Comment:
can you see if you can use exec(str) method in TestUtils.

Reply:
changed to exec